src/utilities/_hypre_utilities.h

```diff
@@ -2893,6 +2893,7 @@ hypre_UnorderedIntMapFindCloserFreeBucket( hypre_UnorderedIntMap  *m,
                                            hypre_HopscotchBucket **free_bucket,
                                            HYPRE_Int *free_dist)
 {
+   (void) m; /* Silence "unused parameter" warning */
```

Comment:
Why not use the macro here?

Reply:
Yes, thanks for catching this! I guess it was an early version of everything here...